lib/services/build.ts

```diff
@@ -496,17 +494,30 @@ export class BuildService implements Project.IBuildService {
 			this.$logger.debug("zipping completed, result file size: %s", fileSize.toString());
 			var projectName = this.$project.projectData.ProjectName;
 			var bucketKey = util.format("%s_%s", projectName, path.basename(projectZipFile));
+			commonHelpers.printInfoMessageOnSameLine("Uploading...");
 			if(fileSize > BuildService.CHUNK_UPLOAD_MIN_FILE_SIZE) {
 				this.$logger.trace("Start uploading file by chunks.");
-				this.$multipartUploadService.uploadFileByChunks(projectZipFile, bucketKey).wait();
-				this.$server.projects.importLocalProject(projectName, projectName, bucketKey).wait();
+				this.printDotsWhileWaiting(this.$multipartUploadService.uploadFileByChunks(projectZipFile, bucketKey), 2000).wait();
+				this.printDotsWhileWaiting(this.$server.projects.importLocalProject(projectName, projectName, bucketKey), 2000).wait();
 			} else {
-				this.$server.projects.importProject(projectName, projectName,
-					this.$fs.createReadStream(projectZipFile)).wait();
+				this.printDotsWhileWaiting(this.$server.projects.importProject(projectName, projectName,
+					this.$fs.createReadStream(projectZipFile)), 2000).wait();
 			}
+
+			commonHelpers.printInfoMessageOnSameLine(os.EOL);
 			this.$logger.trace("Project imported");
 		}).future<void>()();
 	}
+
+	private printDotsWhileWaiting(future: IFuture<any>, timeout: number): IFuture<void> {
```

Comment:
I would change the name to something like "printProgressIndicator" or smthg in the same vein.